packages/fx-core/src/component/coordinator/index.ts

```diff
@@ -265,6 +269,13 @@ class Coordinator {
         if (meArchitecture) {
           feature = `${feature}:${meArchitecture}`;
         }
+        if (
+          (capability === CapabilityOptions.nonSsoTab().id ||
+            capability === CapabilityOptions.tab().id) &&
+          inputs["targetFramework"] === "net8.0"
```

Comment:
Use `inputs.targetFramework` instead. 
And for better readability, I prefer:
```
inputs.targetFramework === "net8.0" && 
(capability === CapabilityOptions.nonSsoTab().id || capability === CapabilityOptions.tab().id)
```

Reply:
Ok, I'll make the .net judgment the first judgment.